### PR TITLE
Add Slibro

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**
## What does this PR do?
Added a new project which was built in Appwrite x Dev Hackathon.
It was a winner 🏆 

## Test Plan

NA

## Related PRs and Issues

NA

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by MrKrishnaAgarwal** [comment]
cc @adityaoberai and @Haimantika

**Comment by MrKrishnaAgarwal** [comment]
More Details abou the project: https://dev.to/sbis04/slibro-a-story-writing-platform-4oll

File: README.md [hunk 1]
```diff
@@ -237,6 +237,8 @@ Share your apps here! Submit a pull request!
 
 * **Quit Smoking App with Flutter and Appwrite** - A simple Counting app with user auth  [Source Code](https://github.com/Jonathanvanhaaften/flutter_appwrite_1)
 
+* **Slibro with Flutter and Appwrite** - A story writing platform that supports short and long format stories. [Source Code](https://github.com/sbis04/slibro)
```
Comment (Meldiron) [inline README.md:240]:
```suggestion
* **Slibro with Flutter and Appwrite** - A story writing platform that supports short and long format stories. [Source Code](https://github.com/sbis04/slibro) [Blog Post](https://dev.to/sbis04/slibro-a-story-writing-platform-4oll)
```

Let's also link the blog post.

**Comment by MrKrishnaAgarwal** [comment]
Done 👍🏻 @Meldiron

**Review by Meldiron — APPROVED** [review]
Looks good to me 🟢 Forwarded the PR to our DevRel team to see if we should include the Hackoberfest label or not before merging.

**Comment by MrKrishnaAgarwal** [comment]
> Looks good to me 

Thanks a lot 🙂

**Comment by adityaoberai** [comment]
@MrKrishnaAgarwal just wanted to confirm if you were involved in the creation of the project? I was looking at their repository but I don't see you listed there as a contributor.

**Comment by MrKrishnaAgarwal** [comment]
> @MrKrishnaAgarwal just wanted to confirm if you were involved in the creation of the project? I was looking at their repository but I don't see you listed there as a contributor.

I didn't contribute to the repo

**Comment by adityaoberai** [comment]
> > @MrKrishnaAgarwal just wanted to confirm if you were involved in the creation of the project? I was looking at their repository but I don't see you listed there as a contributor.
> 
> I didn't contribute to the repo

@MrKrishnaAgarwal definitely appreciate that you shared the project here, however we would prefer if the original creator of the project would add it to this repository. 

We hope you'll check out some of our other issues on the [main Appwrite repository](https://github.com/appwrite/appwrite) and contribute to those :)

**Comment by MrKrishnaAgarwal** [comment]
@adityaoberai most of the non-code issues are assigned, please create more non-code issues

**Pull request closed without merging**